app/src/main/java/com/example/sopt_assignment_dabin/Repository/RepositoryRecyclerView.kt

```diff
@@ -0,0 +1,61 @@
+package com.example.sopt_assignment_dabin
+
+import HorizontalItemDecorator
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.example.sopt_assignment_dabin.Repository.RepositoryListData
+import com.example.sopt_assignment_dabin.Repository.RepositoryRecyclerViewAdapter
+import com.example.sopt_assignment_dabin.databinding.FragmentRepositoryRecyclerViewBinding
+
+class RepositoryRecyclerView : Fragment() {
+    private var _binding: FragmentRepositoryRecyclerViewBinding? = null
+    private val binding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentRepositoryRecyclerViewBinding.inflate(layoutInflater, container, false)
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initRepositoryAdapter()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        _binding = null
+    }
+
+    fun initRepositoryAdapter() {
+        val repositoryAdapter = RepositoryRecyclerViewAdapter()
+        binding.container.adapter = repositoryAdapter
+        binding.container.layoutManager = LinearLayoutManager(requireActivity())  //activity, RecyclerView.VERTICAL, false
```

Comment:
```suggestion
        binding.container.layoutManager = LinearLayoutManager(requireContext()) 
```

requireContext선에서 해결될 수 있습니다.

Reply:
오오 그렇군요 !!